test/test.c

```diff
@@ -5466,6 +5466,31 @@ test_natsSrvVersionAtLeast(void)
     natsConnection_Destroy(nc);
 }
 
+static void
+test_natsFormatStringArray(void)
+{
+    natsStatus s;
+    char *str = NULL;
+
+    test("Check empty: ");
+    s = nats_formatStringArray(&str, NULL, 0);
+    testCond((s == NATS_OK) && (str != NULL) && (strcmp(str, "[]") == 0));
+    NATS_FREE(str);
+    str = NULL;
+
+    test("Check one: ");
+    const char *oneArray[] = {"one"};
+    s = nats_formatStringArray(&str, oneArray, 1);
+    testCond((s == NATS_OK) && (str != NULL) && (strcmp(str, "[\"one\"]") == 0));
+    NATS_FREE(str);
+
```

Comment:
Would set `str = NULL;` after the free above.

Reply:
just changed it to also cover NATS_FREE_STRINGS, so N/A